packages/components/psammead-bulletin/src/index.test.jsx

```diff
@@ -31,6 +39,8 @@ const BulletinComponent = ({ script, service, isLive, mediaType, ctaText }) => {
       ctaText={playCtaText}
       isLive={isLive}
       offScreenText={offScreenText}
+      dir={dir}
+      lang={lang}
```

Comment:
```suggestion

```

Reply:
The intention here is to test for the right output when the `lang` prop is passed in

Reply:
Of course, we need it for the other snapshots https://github.com/bbc/psammead/pull/3430/files#diff-5ae3e98c9edd526e83ba2518475fdcafR67 👍